### PR TITLE
simplify and enhance upload script for docs-tidb-operator pdf

### PR DESCRIPTION
**Pull request by Oreoxmt**

### CODE REVIEW

**Comment by Oreoxmt** [comment]
cc @ran-huang

File: docs-tidb-operator/docs-tidb-operator-merge-pipeline.yaml [hunk 1]
```diff
@@ -21,6 +21,9 @@ tasks:
           if [ "${TARGET_BRANCH}" = "master" ]; then
             python3 scripts/upload.py output_en.pdf tidb-in-kubernetes-dev-en-manual.pdf;
             python3 scripts/upload.py output_zh.pdf tidb-in-kubernetes-dev-zh-manual.pdf;
+          elif [ "${TARGET_BRANCH}" = "release-1.5" ]; then
```
Comment (ran-huang) [inline docs-tidb-operator/docs-tidb-operator-merge-pipeline.yaml:24]: How about this? Consistent with https://github.com/PingCAP-QE/devops-config/blob/15ffab99f85380741e55046571e943d152d68ccc/docs-cn/docs-cn-merge-pipeline.yaml#LL23C11-L27C13
```
          elif [ "${TARGET_BRANCH}" = "release-1.4" ]; then
            python3 scripts/upload.py output.pdf tidb-in-kubernetes-stable-en-manual.pdf;
            python3 scripts/upload.py output.pdf tidb-in-kubernetes-stable-zh-manual.pdf;
          elif case "${TARGET_BRANCH}" in release-*) ;; *) false;; esac; then
            python3 scripts/upload.py output.pdf tidb-in-kubernetes-v${TARGET_BRANCH##*-}-en-manual.pdf;
            python3 scripts/upload.py output.pdf tidb-in-kubernetes-v${TARGET_BRANCH##*-}-zh-manual.pdf;
          fi
```

Comment (Oreoxmt) [reply docs-tidb-operator/docs-tidb-operator-merge-pipeline.yaml]: Done https://github.com/PingCAP-QE/devops-config/pull/152/commits/56d5b098571853906f76239cee34c9da2400929d

**Comment by Oreoxmt** [comment]
@purelind PTAL

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**